src/App.js

```diff
@@ -0,0 +1,29 @@
+import { Provider } from 'react-redux';
+import { Routes, Route, Navigate } from 'react-router-dom';
+import store from './redux/store';
+import './App.css';
+import Sidebar from './components/Sidebar/index';
+import AirPollution from './pages/AirPollution';
+import Contact from './pages/Contact';
+import Dashboard from './pages/Dashboard';
+import Detail from './pages/Detail';
+
+function App() {
```

Comment:
- [ ] The app should follow the design guidelines including the general layout, kindly make sure to implement this so your app looks like the provided design

|Your app|Provided design|
|-|-|
|![Screenshot_20230107_002401](https://user-images.githubusercontent.com/25227022/211115853-ecb3f750-68fd-43fb-9deb-b9e598ea0be1.png)|![Screenshot_20230107_000520](https://user-images.githubusercontent.com/25227022/211115868-5165240b-ede0-413e-bb66-a00ae69eb03d.png)|

Reply:
It is not possible actually to follow exactly the same exact design. But I followed the pattern:

- [x] User is able to filter the results ( For my API, I don't have a whole bunch of cities)
- [x] When the user clicks on specific card, it goes to details page
- [x] In details page, you can go back to home page

Thank you for understanding